scripts/ddcctl.sh

```diff
@@ -4,31 +4,108 @@
 hp="ddcctl -d 1"
 len="ddcctl -d 2"
 
+poweroff() {
+    # Power button will need pressing to power back on
+    $hp -p 5
+}
+
+volmute() {
+    # Set volume to 0 / mute
+    newvol=0
+    volume=$newvol
+    $hp -v $newvol
+}
+
+voldown() {
+    # Decrement the volume by one
+    newvol=$((volume-1))
+    # But dont be negative
+    [[ $newvol -lt 0 ]] && newvol=0
+    volume=$newvol
+    $hp -v $newvol
+}
+
+volup() {
+    # Increment the volume by one
+    newvol=$((volume+1))
+    # But cap at 30 so we dont damage anything
+    [[ $newvol -gt 30 ]] && newvol=30
+    volume=$newvol
+    $hp -v $volume
+}
+
 dim() {
-	$hp -b 42 -c 26
-	$len -b 4 -c 9
+    $hp -b 42 -c 26
+    $len -b 4 -c 9
 }
 
 bright() {
-	$hp -b 100 -c 75
-	$len -b 85 -c 80
+    $hp -b 100 -c 75
+    $len -b 85 -c 80
 }
 
 up() {
-	$hp -b 20+ -c 12+
-	$len -b 15+ -c 12+
```

Comment:
I guess the issue is that your monitor(s) don't report current values wit `ddcctl` correctly but _does_ accept new ones from it?